docs/why.qmd

```diff
@@ -28,43 +228,58 @@ and robust framework for data manipulation in Python.
 In the long-term, we aim for a standard query plan Intermediate Representation
 (IR) like [Substrait](https://substrait.io) to simplify this further.
 
-## Ecosystem
+## Scaling up and out
 
-Ibis is part of a larger ecosystem of Python data tools.
+Out of the box, Ibis offers a great local experience for working with many file
+formats. You can scale up with DuckDB (the default backend) or choose from other
+great options like Polars and DataFusion to work locally with large datasets.
+Once you hit scaling issues on a local machine, you can continue scaling up with
+a larger machine in the cloud using the same backend and same code.
 
-It is designed to work well with other tools in this ecosystem, and we continue
-to make it easier to use Ibis with other tools over time.
+If you hit scaling issues on a large single-node machine, you can switch to a
+distributed backend like PySpark, BigQuery, or Trino by simply changing your
+connection string.
 
-### Local experience
+## Stream-batch unification
 
-Out of the box, Ibis offers a great local experience for working with many file
-formats.
+As of [Ibis 8.0](./posts/ibis-version-8.0.0-release/index.qmd), the first stream
+processing backends have been added. Since these systems tend to support SQL, we
```

Comment:
We can describe why unification is possible with a single Python dataframe API. The concept is stream-table duality. https://www.confluent.io/blog/kafka-streams-tables-part-1-event-streaming/